isaric/parsers/ecmo_subject.toml

```diff
@@ -0,0 +1,229 @@
+    field = "sex"
+    description = "Sex at Birth"
+    values = { 1 = "male", 2 = "female", 3 = "non_binary" }
+
+  [subject.subject_id]
+    field = "\ufeffsubjid"
+    sensitive = true
+    description = "Participant Identification Number (PIN) specify CPMS Site code (hyphen) four to six digit number patient number e.g. Y0401-0001."
+
+  [subject.enrolment_date]
+    field = "dsstdat"
+    description = "Date of Enrolment"
+
+  [subject.admission_date]
+    field = "hostdat"
+    description = "Admission date at this facility"
+
+  [subject.ethnicity] # TODO: need to add ethnic other field
+    description = "Ethnic groups"
+    combinedType = "list"
+    excludeWhen = "none"
+    fields = [
+      { field = "ethnic___1", values = { 1 = "Arab" } },
+      { field = "ethnic___2", values = { 1 = "Black" } },
+      { field = "ethnic___3", values = { 1 = "East_Asian" } },
+      { field = "ethnic___4", values = { 1 = "South_Asian" } },
+      { field = "ethnic___5", values = { 1 = "West_Asian" } },
+      { field = "ethnic___6", values = { 1 = "Latin_American" } },
+      { field = "ethnic___7", values = { 1 = "White" } },
+      { field = "ethnic___8", values = { 1 = "Aboriginal_First_Nations" } },
+    ]
+
+  [subject.works_microbiology_lab]
+    field = "labwork_erterm"
+    ref = "Y/N/NK"
+    description = "Employed in a microbiology laboratory?"
+
+  [subject.works_healthcare]
+    field = "healthwork_erterm"
+    ref = "Y/N/NK"
+    description = "Employed as a healthcare worker?"
+
+  [subject.age]
+    combinedType = "firstNonNull"
+    [[subject.age.fields]]
+      field = "age_estimate10"
+      unit = "years"
+      description = "Calculated age in years"
+
+    [[subject.age.fields]]
+      field = "age_estimateyears"
+      unit = "years"
+      description = "Age/Estimated age"
+
+      [subject.age.fields.source_unit]
+        field = "age_estimateyearsu"
+        values = { 1 = "months", 2 = "years", 3 = "days" }	
+
+[subject.pregnancy]
+  field = "pregyn_rptestcd"
+  ref = "Y/N/NK"
+  description = "Pregnant?"
+
+[subject.pregnancy_date_of_delivery]
+  field = "dlvrdtc_rptestcd"
+  description = "Delivery date"
+
+[subject.pregnancy_birth_weight_kg]
+  field = "apvs_weight"
+  unit = "kg"
+  description = "Birth weight"
+  source_unit = { field = "apvs_weightu", values = { 1 = "kg", 2 = "lbs" } }
+
+[subject.pregnancy_outcome]
+  field = "pregout_rptestcd"
+  description = "Pregnancy Outcome"
+  values = { 1 = "Live birth", 2 = "Still birth" }
+
+[subject.pregnancy_gestational_outcome]
+  field = "apsc_gestout"
+  description = "Gestational outcome"
+  values = { 1 = "Term birth", 2 = "Preterm birth" }
+
+[subject.pregnancy_whether_breastfed] # SK to check here!!
+  combinedType = "any"
+  description = "Breastfed"
+  fields = [
+    { field = "apsc_brfedind", ref = "Y/N/NK" },
+    { field = "apsc_brfed", values = { 1 = "YES - currently breastfeeding", 2 = "YES - breastfeeding discontinued", 0 = "NO" } }
```

Comment:
field is boolean, so values should map to true/false `values = { 1 = true, 2 = true, 0 = false }`

Reply:
Thanks. I will correct this.